votes/2022-05-18-0.json

```
@@ -0,0 +1,77 @@
+{
+  "description": "Which core module should \"host\" the args parser API?",
+  "outcome": {
+    "winner(s)": [
+      "`node:util`"
+    ]
+  },
+  "votes": {
+    "aduh95": {
+      "`node:process`": 0,
+      "`node:util`": 0
+    },
+    "apapirovski": {
+      "`node:process`": 1,
+      "`node:util`": 0
+    },
+    "BethGriggs": {
+      "`node:process`": 0,
+      "`node:util`": 1
+    },
+    "cjihrig": {
+      "`node:process`": 0,
+      "`node:util`": 0
+    },
+    "fhinkel": {
+      "`node:process`": 0,
+      "`node:util`": 0
+    },
+    "gireeshpunathil": {
+      "`node:process`": 0,
+      "`node:util`": 1
+    },
+    "jasnell": {
+      "`node:process`": 0,
+      "`node:util`": 1
+    },
+    "joyeecheung": {
+      "`node:process`": 0,
+      "`node:util`": 1
+    },
+    "mcollina": {
+      "`node:process`": -100,
+      "`node:util`": 1000
+    },
+    "mhdawson": {
```

Unchange my duplicate change
```suggestion
    "mhdawson": {
      "`node:process`": 0,
      "`node:util`": 1
    },